grains/example.rb

```diff
@@ -1,10 +1,10 @@
 class Grains
-  def square(number)
+  def self.square(number)
     2**(number - 1)
   end
 
-  def total
-    @total ||= (1..64).inject(0) do |sum, number|
+  def self.total
+    @@total ||= (1..64).inject(0) do |sum, number|
```

Comment:
I know this is just an example, but I would like to avoid using class instance variables. These are an invitation to create race conditions, and I feel like they should be avoided.

There really is no reason to save state at all in this exercise:


```
def self.square(number)
  1 << (number-1)
end

def self.total
  square(65) - 1
end
```